internal/resources/webform/webform.js

```diff
@@ -2528,26 +2582,34 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug, headers)
         });
     }
 
-    const loadHistoryItem = (index) => {
+    const loadRequest = (request) => {
         const t = $("#grpc-request-response");
         t.tabs("option", "active", 0);
-        const item = history[index];
-        $("#grpc-request-timeout input").val(item.timeout);
```

Comment:
Oh, looks like you fixed a bug here too!

Reply:
:)